src/functionDesc.ts

```diff
@@ -72,7 +72,7 @@ export function isPartialOverlap(a: FunctionDesc, b: FunctionDesc): boolean {
 /**
  * @ignore Not exported outside of package
  */
-export function compareFunction(a: FunctionDesc, b: FunctionDesc) {
+export function compareFunction(a: FunctionDesc, b: FunctionDesc): -1 | 0 | 1 {
```

Comment:
Are these return type annotations matter of taste or required by the upgrade?

Reply:
They're as a result of upgrading typescript-eslint which is kind-of necessitated by the TypeScript upgrade (typescript-eslint versions support a finite range of TypeScript versions).

That upgrade resulted in either a change or upgrade to either our rules of the default rules which now wants explicit return types on exported parts of a module's interface.

No taste in this PR 😁